text/0000-ibm-liberty-buildpack.md

```diff
@@ -0,0 +1,70 @@
+# Paketo Community Liberty Buildpack
+
+## Summary
+
+1. Create a Cloud Native Buildpack for the purpose of installing the [Open Liberty](https://openliberty.io/) Java runtime. 
+2. Create a new top-level multi buildpack that packages up Eclipse OpenJ9 & Open Liberty for use by application developers.
+
+## Motivation
+
+A [ibm-websphere-liberty-buildpack](https://github.com/cloudfoundry/ibm-websphere-liberty-buildpack) exists in the cloudfoundry community and is widely used and popular.  This RFC will
+take that buildpack to the next level, cloud native.  This buildpack will eventually replace the v2 CF ibm-websphere-buildpack.   
+
+## Detailed Explanation
+
+### Open Liberty Buildpack
+
+This buildpack has the purpose of installing the Open Liberty. It will will be based on the Paketo [apache-tomcat](https://github.com/paketo-buildpacks/apache-tomcat) buildpack. 
+
+This buildpack will participate if all the following conditions are met:
+
+- `<APPLICATION_ROOT>/WEB-INF` exists or
+-  A liberty server directory exists
+      a server.xml exists in the --path  or
+-  A liberty packaged server exists 
+	  an unarchived zip file exists containing a wlp/usr/servers/*/server.xml in the --path and
+- `Main-Class` is NOT defined in the mainfest
```

Comment:
```suggestion
- `<APPLICATION_ROOT>/WEB-INF` exists or
-  `<APPLICATION_ROOT>/server.xml` exists or
-  `<APPLICATION_ROOT>/wlp/usr/servers/*/server.xml` exists and
- `Main-Class` is NOT defined in the manifest
```
Is this what is meant?

When a directory or archive is passed to pack with `--path` it is recursively copied to or expanded to the application directory (which is referred to as `<APPLICATION_ROOT>` is the java buildpack READMEs).